context_test.go

```diff
@@ -32,34 +30,3 @@ func TestContext(t *testing.T) {
 	assert.Equal(t, "one", ctx.Value(1))
 	assert.Equal(t, "two point two", ctx.Value(2.2))
 }
-
-func TestWithContext(t *testing.T) {
```

Comment:
This test is no effect, it will panic, because `c.instance.Request` is nil. So remove it directly, test the logic in `middleware_timeout_test.go`.